src/guides/v2.3/get-started/authentication/gs-authentication.md

```diff
@@ -165,6 +159,43 @@ Each type of client has a preferred authentication method. To authenticate, use
    </tr>
 </table>
 
+The following sections describe tips about which type of authentication method you should use depending on your use case.
+
+### Token based (Bearer Authentication)
+
+This method is a good choice for authenticating customers and Admin users in third-party applications that need to make authorized API calls to the Magento store.
+
+*  **Customer Token**—Use this token in applications to authorize specific customer and query data related to that customer (for example, customer details, cart, and orders).
+*  **Admin Token**—Use this token in applications to authorize an Admin user and access Admin-related APIs.
+
+[Request a token]({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#request-token) and then (include it in future requests)({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#web-api-access).
+
+{:.bs-callout-info}
+You should use this type of authentication mechanism over HTTPS:
+
+### Integration (Bearer Authentication)
+
+This method is a good choice for integrating with a third-party system that supports this kind of authentication. You can restrict access to specific resources.
+
+Magento generates a consumer key, consumer secret, access token, and access token secret when you create an active integration (self activated).
+
+To use bearer authentication for API requests, you can use an access token. [Create an active integration]({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#integration-tokens) (self activated) and [use the access token]({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#web-api-access) in the authentication header:
```

Comment:
Hi @jeff-matthews. Very nice explanation! May I add a minor suggestion regarding the authentication header. I would rather name it as "Authorization header" or "Authorization request header" than "Authentication header". The authentication header has a little bit different meaning in the network architecture. In the case of web requests, you usually see such a definition as authorization header.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

But it's not a big deal. The meaning is more or less the same after all.

```suggestion
To use bearer authentication for API requests, you can use an access token. [Create an active integration]({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#integration-tokens) (self activated) and [use the access token]({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#web-api-access) in the authorization header:
```